ABB.SrcML/LibSrcMLRunner.cs

```diff
@@ -99,9 +105,12 @@ public struct SourceData : IDisposable {
             private int bufferCount;
             private IntPtr bufferSize;
             private int tabstop;
-            private ulong optionset;
-            private ulong optionenable;
-            private ulong optiondisable;
+            [MarshalAs(UnmanagedType.U4)]
+            private UInt32 optionset;
```

Comment:
Private fields should be in camelCase. https://github.com/abb-iss/SrcML.NET/wiki/Coding-Standards